plugins/tableresize/plugin.js

```diff
@@ -70,7 +74,12 @@
 		}
 
 		if ( table.$.tFoot ) {
-			pillarHeight += table.$.tFoot.offsetHeight;
+			var tFoot = new CKEDITOR.dom.element( table.$.tFoot );
```

Comment:
After that changes we have very similar code in three parts of this plugin. It will be nice to extract it into separate function.